{"error":{"code":111,"message":"schema mismatch for Product.cape; expected File but got String"},"functionName":"publish-cape-story","level":"error","message":"Failed running cloud function publish-cape-story for user mWkOjox3Mn with:\n  Input: {\"user\":\"mWkOjox3Mn\",\"title\":\"tt\",\"cape\":\"\",\"description\":\"vvvv\",\"category\":\"PnLsigoeJL\"}\n  Error: {\"message\":\"schema mismatch for Product.cape; expected File but got String\",\"code\":111}","params":{"cape":"","category":"PnLsigoeJL","description":"vvvv","title":"tt","user":"mWkOjox3Mn"},"timestamp":"2024-03-08T22:46:13.576Z","user":"mWkOjox3Mn"}
{"code":111,"level":"error","message":"schema mismatch for Product.cape; expected File but got String","stack":"Error: schema mismatch for Product.cape; expected File but got String\n    at SchemaController.enforceFieldExists (C:\\Users\\fabio\\AppData\\Roaming\\npm\\node_modules\\parse-server\\lib\\Controllers\\SchemaController.js:1118:15)\n    at SchemaController.validateObject (C:\\Users\\fabio\\AppData\\Roaming\\npm\\node_modules\\parse-server\\lib\\Controllers\\SchemaController.js:1249:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-03-08T22:46:13.589Z"}
{"error":{"code":111,"message":"schema mismatch for Product.cape; expected File but got String"},"functionName":"publish-cape-story","level":"error","message":"Failed running cloud function publish-cape-story for user mWkOjox3Mn with:\n  Input: {\"user\":\"mWkOjox3Mn\",\"title\":\"tt\",\"cape\":\"\",\"description\":\"vvvv\",\"category\":\"PnLsigoeJL\"}\n  Error: {\"message\":\"schema mismatch for Product.cape; expected File but got String\",\"code\":111}","params":{"cape":"","category":"PnLsigoeJL","description":"vvvv","title":"tt","user":"mWkOjox3Mn"},"timestamp":"2024-03-08T22:46:49.877Z","user":"mWkOjox3Mn"}
{"code":111,"level":"error","message":"schema mismatch for Product.cape; expected File but got String","stack":"Error: schema mismatch for Product.cape; expected File but got String\n    at SchemaController.enforceFieldExists (C:\\Users\\fabio\\AppData\\Roaming\\npm\\node_modules\\parse-server\\lib\\Controllers\\SchemaController.js:1118:15)\n    at SchemaController.validateObject (C:\\Users\\fabio\\AppData\\Roaming\\npm\\node_modules\\parse-server\\lib\\Controllers\\SchemaController.js:1249:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-03-08T22:46:49.882Z"}
{"error":{"code":111,"message":"schema mismatch for Product.cape; expected File but got String"},"functionName":"publish-cape-story","level":"error","message":"Failed running cloud function publish-cape-story for user mWkOjox3Mn with:\n  Input: {\"user\":\"mWkOjox3Mn\",\"title\":\"ff\",\"cape\":\"\",\"description\":\"ff\",\"category\":\"PnLsigoeJL\"}\n  Error: {\"message\":\"schema mismatch for Product.cape; expected File but got String\",\"code\":111}","params":{"cape":"","category":"PnLsigoeJL","description":"ff","title":"ff","user":"mWkOjox3Mn"},"timestamp":"2024-03-08T22:51:17.180Z","user":"mWkOjox3Mn"}
{"code":111,"level":"error","message":"schema mismatch for Product.cape; expected File but got String","stack":"Error: schema mismatch for Product.cape; expected File but got String\n    at SchemaController.enforceFieldExists (C:\\Users\\fabio\\AppData\\Roaming\\npm\\node_modules\\parse-server\\lib\\Controllers\\SchemaController.js:1118:15)\n    at SchemaController.validateObject (C:\\Users\\fabio\\AppData\\Roaming\\npm\\node_modules\\parse-server\\lib\\Controllers\\SchemaController.js:1249:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-03-08T22:51:17.184Z"}
{"error":{"code":141,"message":"INVALID_CREDENTIALS"},"functionName":"login","level":"error","message":"Failed running cloud function login for user undefined with:\n  Input: {\"email\":\"fs894488@gmail.com\",\"password\":\"********\"}\n  Error: {\"message\":\"INVALID_CREDENTIALS\",\"code\":141}","params":{"email":"fs894488@gmail.com","password":"********"},"timestamp":"2024-03-08T23:23:07.810Z"}
{"code":141,"level":"error","message":"INVALID_CREDENTIALS","stack":"Error: INVALID_CREDENTIALS\n    at Object.resolveError (C:\\Users\\fabio\\AppData\\Roaming\\npm\\node_modules\\parse-server\\lib\\triggers.js:531:12)\n    at error (C:\\Users\\fabio\\AppData\\Roaming\\npm\\node_modules\\parse-server\\lib\\Routers\\FunctionsRouter.js:99:32)","timestamp":"2024-03-08T23:23:07.814Z"}
